web/api/netdata-swagger.json

```diff
@@ -156,7 +167,7 @@
           {
             "name": "after",
             "in": "query",
-            "description": "This parameter can either be an absolute timestamp specifying the starting point of the data to be returned, or a relative number of seconds (negative, relative to parameter: before). Netdata will assume it is a relative number if it is less that 3 years (in seconds). Netdata will adapt this parameter to the boundaries of the round robin database. The default is the beginning of the round robin database (i.e. by default netdata will attempt to return data for the entire database).",
+            "description": "This parameter can either be an absolute timestamp specifying the starting point of the data to be returned, or a relative number of seconds (negative, relative to parameter: before). Netdata will assume it is a relative number if it is less that 3 years (in seconds). If not specified the default is -600 seconds. Netdata will adapt this parameter to the boundaries of the round robin database unless the allow_past option is specified.",
```

Comment:
How is the default -600? I thought it was still the beginning of the database.

Reply:
This changed back in March 2020 in the code.  The docs specified a default of -600, the code didn't.

Reply:
So then I guess we have to update README.md as well then?